src/Symfony/Component/Mime/Resources/bin/update_mime_types.php

```diff
@@ -21,7 +21,22 @@
     $new[$mimeType] = $extensions;
 }
 
-$xml = simplexml_load_string(file_get_contents('https://raw.github.com/minad/mimemagic/master/script/freedesktop.org.xml'));
```

Comment:
This file is not available anymore upstream, let's use the one locally available.

Reply:
this makes it harder to know which version of freedesktop.org.xml was used to generate the list (not all systems will have the latest one, so someone running the script might actually downgrade the list).
Wouldn't it be better to download the file from its official source ?

Reply:
But someone would always be me :)

Reply:
would it make sense to compile the file mtime rather than the current date?  to verify we dont go back in time.

Reply:
Updated to a better/uptodate source